uml/sequence_diagrams/ClockView.txt

```diff
@@ -0,0 +1,46 @@
+@startuml
+participant DeactivatedClock 
+participant ClockView 
+participant BlueTooth
+participant Battery
+participant DisplayError 
+participant DisplayViews 

```

Comment:
DisplayViews is the same as DisplaySelection in my display.png ?

If so, it might be worth aligning on names

Reply:
I will update mine

Reply:
fixed

Reply:
I think it also needs to be changed in the other files:

DisplayViewSequence - DisplayViews still present but unused - can be removed

also : 
Map, Settings - DisplayViews should change to DisplaySelection

Reply:
Fixed